packages/components/app/styles/components/tag.scss

```diff
@@ -29,7 +29,7 @@ $hds-tag-border-radius: 50px;
 .hds-tag__dismiss {
   flex: 0 0 auto;
   margin: 0; // reset default button margin
-  padding: 6px 2px 6px 8px;
+  padding: 6px 4px 6px 8px;
```

Comment:
This would need a changelog entry

Reply:
changes like this should be cherry-picked out into a separate PR

Reply:
1. There is a changeset
2. I left it here because I felt it was directly related and a tiny change
3. I'll pull it out into a separate PR.

Reply:
From my perspective, this will make it easier to land that change more quickly/cleanly than tying it up in this PR which is more of an internally facing change that has a bunch of threads in it.

Reply:
I wouldn't have reverted the change here, but rather rebase this on top of the other PR/main once it is merged so that the fix is here but can get upstreamed faster.